### PR TITLE
TiDB Cloud CLI New

### PR DESCRIPTION
**Pull request by shiyuhang0**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md
Comment (qiancai) [inline TOC-tidb-cloud.md:1]: preview of this PR: https://pingcap-docsite-preview-git-pr-qiancai.vercel.app/tidbcloud/cli-reference

File: tidb-cloud/cli-reference.md [hunk 1]
```diff
@@ -17,17 +17,19 @@ The following table lists the commands available for the TiDB Cloud CLI.
...
+| serverless export  | create, describe, list, cancel, download                                 | Manage TiDB Serverless export jobs                                                                       |
+| serverless import  | cancel, describe, list, start                                            | Manage [import](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) tasks |
+| auth               | login, logout                                                            | Login and logout                                                                                         |
+| ai                 | -                                                                        | Chat with TiDB Bot                                                                                       |
+| completion         | bash, fish, powershell, zsh                                              | Generate completion script for specified shell                                                           |
```
Comment (qiancai) [inline tidb-cloud/cli-reference.md:28]: Does it need to be listed in the TOC-tidb-cloud.md?

File: tidb-cloud/cli-reference.md [hunk 2]
```diff
@@ -17,17 +17,19 @@ The following table lists the commands available for the TiDB Cloud CLI.
...
+| config             | create, delete, describe, edit, list, set, use                           | Configure user profiles                                                                                  |
+| help               | cluster, completion, config, help, import, project, update               | View help for any command                                                                                |
```
Comment (qiancai) [inline tidb-cloud/cli-reference.md:30]: Does it need to be listed in the TOC-tidb-cloud.md?

File: tidb-cloud/ticloud-auth-login.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Examples
+
+To start the login for your account:
+
+```shell
+ticloud auht login
+```
+
+Login with insecure storage:
```
Comment (qiancai) [inline tidb-cloud/ticloud-auth-login.md:22]:
````suggestion
To log into TiDB Cloud CLI:

```shell
ticloud auth login
```

To log into TiDB Cloud CLI with insecure storage:
````

File: tidb-cloud/ticloud-auth-login.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+```shell
+ticloud auht login --insecure-storage
```
Comment (qiancai) [inline tidb-cloud/ticloud-auth-login.md:25]:
```suggestion
ticloud auth login --insecure-storage
```

File: tidb-cloud/ticloud-auth-login.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+title: ticloud auth login
+summary: The reference of `ticloud auth login`.
+---
+
+# ticloud auth login
+
+Authenticate with the TiDB Cloud:
```
Comment (qiancai) [inline tidb-cloud/ticloud-auth-login.md:8]:
```suggestion
Authenticate with TiDB Cloud CLI:
```

File: tidb-cloud/ticloud-auth-login.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+|----------------------|--------------------------------------------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------|
+| --no-color           | Disables color in output.                                                                  | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |
+| -P, --profile string | The active [user profile](/tidb-cloud/cli-reference.md#user-profile) used in this command. | No       | Works in both non-interactive and interactive modes.                                                             |
+| -D, --debug          | Enable debug mode                                                                          | No       | Works in both non-interactive and interactive modes.                                                             |
```
Comment (qiancai) [inline tidb-cloud/ticloud-auth-login.md:43]:
```suggestion
| -D, --debug          | Enable debug mode.                                                                          | No       | Works in both non-interactive and interactive modes.                                                             |
```

File: tidb-cloud/ticloud-auth-logout.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+---
+title: ticloud auth logout
+summary: The reference of `ticloud auth logout`.
+---
+
+# ticloud auth logout
+
+Log out of the CLI:
```
Comment (qiancai) [inline tidb-cloud/ticloud-auth-logout.md:8]:
```suggestion
Log out of TiDB Cloud CLI:
```

File: tidb-cloud/ticloud-auth-logout.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+```shell
+ticloud auth logout [flags]
+```
+
+## Examples
+
+To log out of the TiDB Cloud:
```
Comment (qiancai) [inline tidb-cloud/ticloud-auth-logout.md:16]:
```suggestion
To log out of TiDB Cloud CLI:
```

File: tidb-cloud/ticloud-auth-logout.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+```shell
+ticloud auht logout
```
Comment (qiancai) [inline tidb-cloud/ticloud-auth-logout.md:19]:
```suggestion
ticloud auth logout
```

File: tidb-cloud/ticloud-auth-logout.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+|----------------------|--------------------------------------------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------|
+| --no-color           | Disables color in output.                                                                  | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |
+| -P, --profile string | The active [user profile](/tidb-cloud/cli-reference.md#user-profile) used in this command. | No       | Works in both non-interactive and interactive modes.                                                             |
+| -D, --debug          | Enable debug mode                                                                          | No       | Works in both non-interactive and interactive modes.                                                             |
```
Comment (qiancai) [inline tidb-cloud/ticloud-auth-logout.md:28]:
```suggestion
| -D, --debug          | Enable debug mode.                                                                          | No       | Works in both non-interactive and interactive modes.                                                             |
```

File: tidb-cloud/ticloud-serverless-update.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
+---
+title: ticloud serverless cluster update
+summary: The reference of `ticloud serverless update`.
+---
+
+# ticloud serverless update
+
+Update a serverless cluster:
```
Comment (qiancai) [inline tidb-cloud/ticloud-serverless-update.md:8]:
```suggestion
Update a TiDB Serverless cluster:
```

File: tidb-cloud/ticloud-serverless-update.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+
+```shell
+ticloud serverless update [flags]
+```
+
+## Examples
+
+Update a serverless cluster in interactive mode:
```
Comment (qiancai) [inline tidb-cloud/ticloud-serverless-update.md:16]:
```suggestion
Update a TiDB Serverless cluster in interactive mode:
```

File: tidb-cloud/ticloud-serverless-update.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+
+```shell
+ticloud serverless update
+```
+
+Update displayName of a serverless cluster in non-interactive mode:
```
Comment (qiancai) [inline tidb-cloud/ticloud-serverless-update.md:22]:
```suggestion
Update `displayName` of a TiDB Serverless cluster in non-interactive mode:
```

File: tidb-cloud/ticloud-serverless-update.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+
+```shell
+ticloud serverless update -c <cluster-id> --display-name <new-display-mame>
+```
+
+Update labels of a serverless cluster in non-interactive mode
```
Comment (qiancai) [inline tidb-cloud/ticloud-serverless-update.md:28]:
```suggestion
Update labels of a TiDB Serverless cluster in non-interactive mode
```

File: tidb-cloud/ticloud-serverless-update.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+## Flags
+
+In non-interactive mode, you need to manually enter the required flags. In interactive mode, you can just follow CLI prompts to fill them in.
+
+| Flag                     | Description                                           | Required | Note                                                 |
+|--------------------------|-------------------------------------------------------|----------|------------------------------------------------------|
+| -c, --cluster-id string  | The ID of the cluster to be updated                   | Yes      | Only works in non-interactive mode.                  |
+| -n --display-name string | The new displayName of the cluster to be updated      | No       | Only works in non-interactive mode.                  |
+| --annotations string     | The annotations of the cluster to be added or updated | No       | Only works in non-interactive mode.                  |
+| --labels string          | The labels of the cluster to be added or updated      | No       | Only works in non-interactive mode.                  |
```
Comment (qiancai) [inline tidb-cloud/ticloud-serverless-update.md:43]:
```suggestion
| -c, --cluster-id string  | The new ID of the cluster                   | Yes      | Only works in non-interactive mode.                  |
| -n --display-name string | The new `displayName` of the cluster      | No       | Only works in non-interactive mode.                  |
| --annotations string     | The new annotations of the cluster | No       | Only works in non-interactive mode.                  |
| --labels string          | The new labels of the cluster      | No       | Only works in non-interactive mode.                  |
```

File: tidb-cloud/ticloud-serverless-update.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+|----------------------|------------------------------------------------------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------|
+| --no-color           | Disables color in output.                                                                            | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |
+| -P, --profile string | Specifies the active [user profile](/tidb-cloud/cli-reference.md#user-profile) used in this command. | No       | Works in both non-interactive and interactive modes.                                                             |
+| -D, --debug          | Enable debug mode                                                                                    | No       | Works in both non-interactive and interactive modes.                                                             |
```
Comment (qiancai) [inline tidb-cloud/ticloud-serverless-update.md:52]:
```suggestion
| -D, --debug          | Enable debug mode.                                                                                    | No       | Works in both non-interactive and interactive modes.                                                             |
```

File: TOC-tidb-cloud.md
Comment (qiancai) [inline TOC-tidb-cloud.md:1]: PR preview: https://pingcap-docsite-preview-git-pr-qiancai.vercel.app/tidbcloud/cli-reference

File: tidb-cloud/ticloud-serverless-spending-limit.md [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+
+## Flags
+
+In non-interactive mode, you need to manually enter the required flags. In interactive mode, you can just follow CLI prompts to fill them in.
+
+| Flag                    | Description                                 | Required | Note                                                 |
+|-------------------------|---------------------------------------------|----------|------------------------------------------------------|
+| -c, --cluster-id string | Specifies the ID of the cluster.                       | Yes      | Only works in non-interactive mode.                  |
+| --monthly int32         | Specifies the maximum monthly spending limit in USD cents. | Yes      | Only works in non-interactive mode.                  |
+| --force                 | Deletes a cluster without confirmation.      | No       | Works in both non-interactive and interactive modes. |
```
Comment (qiancai) [inline tidb-cloud/ticloud-serverless-spending-limit.md:36]: "Deletes a cluster without confirmation." looks incorrect here for this command

File: tidb-cloud/ticloud-import-cancel.md [hunk 18]
```diff
@@ -1,48 +1,49 @@
...
-| --force                 | Deletes a profile without confirmation | No       | Works in both non-interactive and interactive modes. |
-| -h, --help              | Help information for this command             | No       | Works in both non-interactive and interactive modes. |
-| --import-id string      | The ID of the import task                 | Yes      | Only works in non-interactive mode.                   |
-| -p, --project-id string | Project ID                            | Yes      | Only works in non-interactive mode.                   |
+| Flag                    | Description                            | Required | Note                                                 |
+|-------------------------|----------------------------------------|----------|------------------------------------------------------|
+| -c, --cluster-id string | Specifies the ID of the cluster.                             | Yes      | Only works in non-interactive mode.                  |
+| --force                 | Deletes a profile without confirmation. | No       | Works in both non-interactive and interactive modes. |
```
Comment (qiancai) [inline tidb-cloud/ticloud-import-cancel.md:35]: "Deletes a profile without confirmation." looks incorrect here for this command

File: tidb-cloud/ticloud-serverless-spending-limit.md (hunk 17 above)
Comment (shiyuhang0) [reply tidb-cloud/ticloud-serverless-spending-limit.md:36]: It shoud be deleted

File: tidb-cloud/ticloud-import-cancel.md (hunk 18 above)
Comment (shiyuhang0) [reply tidb-cloud/ticloud-import-cancel.md:35]: fixed

File: tidb-cloud/cli-reference.md (hunk 2 above)
Comment (shiyuhang0) [reply tidb-cloud/cli-reference.md:30]: I think `help` needn't to be listed

File: TOC-tidb-cloud.md [hunk 19]
```diff
@@ -620,18 +620,40 @@
       - [Spill to Disk](/tiflash/tiflash-spill-disk.md)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:620]: preview: https://pingcap-docsite-preview-git-pr-qiancais-projects.vercel.app/tidbcloud/cli-reference

File: tidb-cloud/ticloud-import-start.md [hunk 20]
```diff
@@ -0,0 +1,80 @@
...
+ticloud serverless import start --local.file-path <file-path> --cluster-id <cluster-id> --file-type <file-type> --local.target-database <target-database> --local.target-table <target-table>
+```
+
+Start a local import task with custom upload concurrency:
+
+```shell
+ticloud serverless import start --local.file-path <file-path> --cluster-id <cluster-id> --file-type <file-type> --local.target-database <target-database> --local.target-table <target-table> --local.concurrency 10
+```
+
+Start an local import task with custom CSV format:
```
Comment (hfxsd) [inline tidb-cloud/ticloud-import-start.md:44]:
```suggestion
Start a local import task with custom CSV format:
```

File: tidb-cloud/ticloud-import-start.md [hunk 21]
```diff
@@ -0,0 +1,80 @@
...
+| --csv.delimiter string         | The delimiter used for quoting of CSV file. (default "\"")                                                                    | No       | Only works in non-interactive mode.                                       |
+| --csv.separator string         | The field separator of CSV file. (default ",")                                                                                | No       | Only works in non-interactive mode.                                       |
+| --csv.trim-last-separator      | In CSV file whether to treat separator as the line terminator and trim all trailing separators.                               | No       | Only works in non-interactive mode.                                       |
+| -h, --help                     | Shows help information for this command.                                                                                      | No       | Works in both non-interactive and interactive modes.                      |
```
Comment (hfxsd) [inline tidb-cloud/ticloud-import-start.md:68]:
```suggestion
| -c, --cluster-id string        | Specifies the cluster ID.                                                                                                     | Yes      | Only works in non-interactive mode.                                       |
| --source-type string           | Specifies the import source type. The default value is `LOCAL`.                                                                   | No       | Only works in non-interactive mode.                                       |
| --local.concurrency int        | Specifies the concurrency for uploading files. The default value is `5`.                                                                               | No       | Only works in non-interactive mode.                                       |
| --local.file-path string       | Specifies the local file path to import.                                                                                                | No       | Only works in non-interactive mode.                                       |
| --local.target-database string | Specifies the target database to which the data is imported.                                                                                         | No       | Only works in non-interactive mode.                                       |
| --local.target-table string    | Specifies the target table to which the data is imported.                                                                                            | No       | Only works in non-interactive mode.                                       |
| --local.file-path string       | Specifies the local file path to import the data.                                                                                                | No       | Only works in non-interactive mode.                                       |
| --file-type string             | Specifies the import file type, one of ["CSV"].                                                                                         | Yes      | Only works in non-interactive mode.                                       |
| --csv.backslash-escape         | Specifies whether to parse backslash inside fields as escape characters in a CSV file. The default value is `true`.                                     | No       | Only works in non-interactive mode.                                       |
| --csv.delimiter string         | Specifies the delimiter used for quoting of a CSV file. The default value is `\`.                                                                    | No       | Only works in non-interactive mode.                                       |
| --csv.separator string         | Specifies the field separator of a CSV file. The default value is `,`                                                                                | No       | Only works in non-interactive mode.                                       |
| --csv.trim-last-separator      | Specifies whether to treat the separator as the line terminator and trim all trailing separators in a CSV file.                               | No       | Only works in non-interactive mode.                                       |
| -h, --help                     | Shows help information for this command.                                                                                      | No       | Works in both non-interactive and interactive modes.                      |
```

File: tidb-cloud/ticloud-serverless-export-create.md [hunk 22]
```diff
@@ -0,0 +1,61 @@
...
+| --s3.bucket-uri string        | Specifies the bucket URI of the S3. This flag is required when you export data to Amazon S3.                                              | No       | Only works in non-interactive mode.                  |
+| --s3.access-key-id string     | Specifies the access key ID of the S3 bucket. This flag is required when you export data to Amazon S3.                                    | NO       | Only works in non-interactive mode.                  |
+| --s3.secret-access-key string | Specifies the secret access key of the S3 bucket. This flag is required when you export data to Amazon S3.                                | No       | Only works in non-interactive mode.                  |
+| --compression string          | Specifies the compression algorithm of the export file. The supported algorithms include `GZIP` `SNAPPY` `ZSTD` `NONE`. The default value is `GZIP`. | No       | Only works in non-interactive mode.                  |
```
Comment (hfxsd) [inline tidb-cloud/ticloud-serverless-export-create.md:48]:
```suggestion
| --compression string          | Specifies the compression algorithm of the export file. The supported algorithms include `GZIP`, `SNAPPY`, `ZSTD`, and `NONE`. The default value is `GZIP`. | No       | Only works in non-interactive mode.                  |
```

File: tidb-cloud/ticloud-serverless-regions.md [hunk 23]
```diff
@@ -0,0 +1,47 @@
+---
+title: ticloud serverless cluster regions
```
Comment (hfxsd) [inline tidb-cloud/ticloud-serverless-regions.md:2]:
```suggestion
title: ticloud serverless regions
```

File: tidb-cloud/ticloud-serverless-shell.md [hunk 24]
```diff
@@ -0,0 +1,61 @@
+---
+title: ticloud serverless cluster shell
```
Comment (hfxsd) [inline tidb-cloud/ticloud-serverless-shell.md:2]:
```suggestion
title: ticloud serverless shell
```

File: tidb-cloud/ticloud-serverless-spending-limit.md [hunk 25]
```diff
@@ -0,0 +1,48 @@
+---
+title: ticloud serverless cluster spending-limit
```
Comment (hfxsd) [inline tidb-cloud/ticloud-serverless-spending-limit.md:2]:
```suggestion
title: ticloud serverless spending-limit
```

File: tidb-cloud/ticloud-serverless-update.md [hunk 26]
```diff
@@ -0,0 +1,56 @@
+---
+title: ticloud serverless cluster update
```
Comment (hfxsd) [inline tidb-cloud/ticloud-serverless-update.md:2]:
```suggestion
title: ticloud serverless update
```

File: tidb-cloud/ticloud-import-start.md [hunk 27]
```diff
@@ -0,0 +1,80 @@
+---
+title: ticloud serverless import start
+summary: The reference of `ticloud serverless import start`.
+---
+
+# ticloud serverless import start local
```
Comment (qiancai) [inline tidb-cloud/ticloud-import-start.md:6]:
```suggestion
# ticloud serverless import start
```

File: tidb-cloud/ticloud-import-start.md [hunk 28]
```diff
@@ -0,0 +1,80 @@
...
+| --local.file-path string       | Specifies the path of the local file to be imported.                                                                                                | No       | Only works in non-interactive mode.                                       |
+| --local.target-database string | Specifies the target database to which the data is imported.                                                                                         | No       | Only works in non-interactive mode.                                       |
+| --local.target-table string    | Specifies the target table to which the data is imported.                                                                                            | No       | Only works in non-interactive mode.                                       |
+| --local.file-path string       | Specifies the local file path to import the data.                                                                                                | No       | Only works in non-interactive mode.                                       |
```
Comment (qiancai) [inline tidb-cloud/ticloud-import-start.md:62]:
```suggestion
```

Comment (qiancai) [reply tidb-cloud/ticloud-import-start.md:62]: dup with L59

File: tidb-cloud/cli-reference.md [hunk 29]
```diff
@@ -17,17 +17,19 @@ The following table lists the commands available for the TiDB Cloud CLI.
...
+|--------------------|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| auth               | login, logout                                                            | Login and logout                                                                                         |
+| serverless         | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                          |
+| serverless branch  | create, delete, describe, list, shell                                    | Manage TiDB Serverless branches                                                                          |
+| serverless import  | cancel, describe, list, start                                            | Manage TiDB Serverless [import](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) tasks |
```
Comment (zhangyangyu) [inline tidb-cloud/cli-reference.md:25]: why this link? I think it's not suitable here.

File: tidb-cloud/ticloud-import-start.md [hunk 30]
```diff
@@ -0,0 +1,80 @@
...
+| --file-type string             | Specifies the import file type, one of ["CSV"].                                                                                         | Yes      | Only works in non-interactive mode.                                       |
```
Comment (qiancai) [inline tidb-cloud/ticloud-import-start.md:63]: 目前只支持 CSV 是吗

File: tidb-cloud/ticloud-import-start.md [hunk 31]
```diff
@@ -0,0 +1,80 @@
+---
+title: ticloud serverless import start
+summary: The reference of `ticloud serverless import start`.
```
Comment (qiancai) [inline tidb-cloud/ticloud-import-start.md:3]:
```suggestion
summary: The reference of `ticloud serverless import start`.
aliases: ['/tidbcloud/ticloud-import-start-local', '/tidbcloud/ticloud-import-start-mysql, '/tidbcloud/ticloud-import-start-s3']
```

File: tidb-cloud/ticloud-import-start.md (hunk 30 above)
Comment (qiancai) [inline tidb-cloud/ticloud-import-start.md:63]:
```suggestion
| --file-type string             | Specifies the import file type. Currently, only `CSV` is supported.                                                                                         | Yes      | Only works in non-interactive mode.                                       |
```

File: tidb-cloud/cli-reference.md [hunk 32]
```diff
@@ -103,10 +105,10 @@ Use [`ticloud config delete`](/tidb-cloud/ticloud-config-delete.md) to delete a
...
-| --no-color           | Disables color in output.                      | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |
-| -P, --profile string | Specifies the active user profile used in this command. | No       | Works in both non-interactive and interactive modes.                                                                      |
+| Flag                 | Description                                             | Required | Note                                                                                                             |
+|----------------------|---------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------|
+| --no-color           | Disables color in output.                               | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |
```
Comment (zhangyangyu) [inline tidb-cloud/cli-reference.md:110]: add `--debug` flag

File: tidb-cloud/ticloud-import-start.md (hunk 30 above)
Comment (shiyuhang0) [reply tidb-cloud/ticloud-import-start.md:63]: yes

File: tidb-cloud/cli-reference.md (hunk 29 above)
Comment (shiyuhang0) [reply tidb-cloud/cli-reference.md:25]: it used to be this link, i will remove the link now

File: tidb-cloud/ticloud-serverless-regions.md [hunk 33]
```diff
@@ -0,0 +1,47 @@
...
+```shell
+ticloud serverless regions [flags]
+```
+
+## Examples
+
+List all available regions for TiDB Serverless clusters:
+
+```shell
+ticloud serverless regions
```
Comment (zhangyangyu) [inline tidb-cloud/ticloud-serverless-regions.md:19]: regions -> region

File: tidb-cloud/cli-reference.md [hunk 34]
```diff
@@ -17,17 +17,19 @@ The following table lists the commands available for the TiDB Cloud CLI.
...
-| project    | list                                                       | Manage projects                                                                                          |
-| update     | -                                                          | Update the CLI to the latest version                                                                     |
+| Command            | Subcommand                                                               | Description                                                                                              |
+|--------------------|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| auth               | login, logout                                                            | Login and logout                                                                                         |
+| serverless         | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                          |
```
Comment (zhangyangyu) [inline tidb-cloud/cli-reference.md]:
```suggestion
| serverless (alias s)   | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                          |
```

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/ticloud-serverless-region.md [hunk 35]
```diff
@@ -1,28 +1,28 @@
 ---
-title: ticloud serverless regions
-summary: The reference of `ticloud serverless regions`.
+title: ticloud serverless region
+summary: The reference of `ticloud serverless region`.
```
Comment (qiancai) [inline tidb-cloud/ticloud-serverless-region.md:3]:
```suggestion
summary: The reference of `ticloud serverless region`.
aliases: ['/tidbcloud/ticloud-serverless-regions']
```

File: tidb-cloud/cli-reference.md [hunk 36]
```diff
@@ -17,19 +17,19 @@ The following table lists the commands available for the TiDB Cloud CLI.
...
-| help               | cluster, completion, config, help, import, project, update               | View help for any command                                                                                |
+| Command           | Subcommand                                                               | Description                                                                                             |
+|-------------------|--------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| auth              | login, logout                                                            | Login and logout                                                                                        |
+| serverless (alias s)     | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                         |
```
Comment (qiancai) [inline tidb-cloud/cli-reference.md:23]:
```suggestion
| serverless (alias: s)     | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                         |
```

File: tidb-cloud/cli-reference.md [hunk 37]
```diff
@@ -45,12 +45,16 @@ The TiDB Cloud CLI provides two modes for some commands for easy use:
 
 ## User profile
 
-For the TiDB Cloud CLI, a user profile is a collection of properties associated with a user, including the profile name, public key, and private key. To use TiDB Cloud CLI, you must create a user profile first.
+For the TiDB Cloud CLI, a user profile is a collection of properties associated with a user, including the profile name, public key, private key and OAuth token. To use TiDB Cloud CLI, you must have a user profile.
```
Comment (qiancai) [inline tidb-cloud/cli-reference.md:48]:
```suggestion
For the TiDB Cloud CLI, a user profile is a collection of properties associated with a user, including the profile name, public key, private key, and OAuth token. To use TiDB Cloud CLI, you must have a user profile.
```

File: tidb-cloud/cli-reference.md [hunk 38]
```diff
@@ -45,12 +45,16 @@ The TiDB Cloud CLI provides two modes for some commands for easy use:
...
 
-### Create a user profile
+### Create a user profile with Open API key
```
Comment (qiancai) [inline tidb-cloud/cli-reference.md:50]:
```suggestion
### Create a user profile with TiDB Cloud API key
```

File: tidb-cloud/cli-reference.md [hunk 39]
```diff
@@ -109,6 +113,7 @@ The following table lists the global flags for the TiDB Cloud CLI.
 |----------------------|---------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------|
 | --no-color           | Disables color in output.                               | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |
 | -P, --profile string | Specifies the active user profile used in this command. | No       | Works in both non-interactive and interactive modes.                                                             |
+| -D, --debug          | Enable debug mode                                       | No       | Works in both non-interactive and interactive modes.:wq                                                          |
```
Comment (qiancai) [inline tidb-cloud/cli-reference.md:116]:
```suggestion
| -D, --debug          | Enable debug mode                                       | No       | Works in both non-interactive and interactive modes.                                                          |
```

File: tidb-cloud/cli-reference.md [hunk 40]
```diff
@@ -43,12 +45,16 @@ The TiDB Cloud CLI provides two modes for some commands for easy use:
 
 ## User profile
 
-For the TiDB Cloud CLI, a user profile is a collection of properties associated with a user, including the profile name, public key, and private key. To use TiDB Cloud CLI, you must create a user profile first.
+For the TiDB Cloud CLI, a user profile is a collection of properties associated with a user, including the profile name, public key, private key and OAuth token. To use TiDB Cloud CLI, you must have a user profile.
 
-### Create a user profile
+### Create a user profile with Open API key
```
Comment (qiancai) [inline tidb-cloud/cli-reference.md:50]:
```suggestion
### Create a user profile with TiDB Cloud API key
```

File: tidb-cloud/ticloud-import-start.md [hunk 41]
```diff
@@ -0,0 +1,80 @@
...
+aliases: ['/tidbcloud/ticloud-import-start-local', '/tidbcloud/ticloud-import-start-mysql, '/tidbcloud/ticloud-import-start-s3']
```
Comment (qiancai) [inline tidb-cloud/ticloud-import-start.md:4]:
```suggestion
aliases: ['/tidbcloud/ticloud-import-start-local','/tidbcloud/ticloud-import-start-mysql','/tidbcloud/ticloud-import-start-s3']
```

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/ticloud-serverless-export-download.md [hunk 42]
```diff
@@ -0,0 +1,50 @@
+---
+title: ticloud serverless export download
+summary: The reference of `ticloud serverless export download`.
+---
+
+# ticloud serverless export describe
```
Comment (hfxsd) [inline tidb-cloud/ticloud-serverless-export-download.md:6]:
```suggestion
# ticloud serverless export download
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/cli-reference.md [hunk 43]
```diff
@@ -17,17 +21,19 @@ The following table lists the commands available for the TiDB Cloud CLI.
...
-| project    | list                                                       | Manage projects                                                                                          |
-| update     | -                                                          | Update the CLI to the latest version                                                                     |
+| Command           | Subcommand                                                               | Description                                                                                             |
+|-------------------|--------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| auth              | login, logout                                                            | Login and logout                                                                                        |
+| serverless (alias: s)     | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                         |
```
Comment (qiancai) [inline tidb-cloud/cli-reference.md:27]:
```suggestion
| serverless (alias: s)     | create, delete, describe, list, update, spending-limit, region, shell   | Manage TiDB Serverless clusters                                                                         |
```

**Pull request merged**